.github/workflows/test.yml

```diff
@@ -159,6 +159,14 @@ jobs:
       - build-static-arm64
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/go/pkg/mod
+            ~/.cache/go-build
+            /tmp/spdx
```

Comment:
where does this path come from?

Reply:
This is hardcoded into the bom tool: 

https://github.com/kubernetes-sigs/bom/blob/436b95e99c9348a8c3ee459cda81daff50f1b28f/pkg/spdx/spdx.go#L93

Reply:
would you add a comment ?

Reply:
Done :+1: